### PR TITLE
Handle newer unsquashfs

### PR DESCRIPTION
**Pull request by stgraber**

### CODE REVIEW

File: shared/archive_linux.go [hunk 1]
```diff
@@ -76,6 +76,40 @@ func Unpack(file string, path string, blockBackend bool, runningInUserns bool, t
...
+
+			// Confirm that all errors are related to character or block devices.
+			found := false
+			for _, line := range strings.Split(runError.Stderr, "\n") {
+				line = strings.TrimSpace(line)
+				if line == "" {
+					continue
+				}
+
+				if !strings.Contains(line, "failed to create block device") {
```
Comment (tomponline) [inline shared/archive_linux.go:95]: @stgraber why are these safe to skip if we cant unpack them? Thanks

Comment (tomponline) [reply shared/archive_linux.go:95]: And could you add a comment here explaining why that is for future reference.

Comment (stgraber) [reply shared/archive_linux.go:95]: Same reason we filter out all /dev/* stuff above in the same function. We don't care about them and can't create them in unpriv containers.

Comment (tomponline) [reply shared/archive_linux.go:95]: @stgraber ah ok, I didn't see that part, but looking now, does that mean this skipping should only be done if `runningInUserns` is true like above? That would help to self-document it too.

**Review by tomponline — APPROVED** [review]

**Pull request merged**